python/beeai_framework/agents/tool_calling/prompts.py

```diff
@@ -47,7 +47,7 @@ class ToolCallingAgentSystemPromptInput(BaseModel):
 # Date and Time
 The current date and time is: {{formatDate}}
 You do not need a tool to get the current Date and Time. Use the information available here.
-""",
+""",  # noqa: E501
```

Comment:
Is there a reason we need this? Does the prompt behave differently if we "wrap" the long line?

Reply:
ok, then this is a good use of noqa then